cpp/tests/geometry/PointCloud.cpp

```diff
@@ -1186,50 +1187,55 @@ TEST(PointCloud, ComputeConvexHull) {
 }
 
 TEST(PointCloud, HiddenPointRemoval) {
+    data::dataset::SamplePCDFragments pcd_fragments;
     geometry::PointCloud pcd;
-    io::ReadPointCloud(utility::GetDataPathCommon("fragment.ply"), pcd);
-    EXPECT_EQ(pcd.points_.size(), 196133);
-    ExpectEQ(pcd.GetMaxBound(), Eigen::Vector3d(3.96609, 2.427476, 2.55859));
-    ExpectEQ(pcd.GetMinBound(), Eigen::Vector3d(0.558594, 0.832031, 0.566637));
+    io::ReadPointCloud(pcd_fragments.path_to_fragments_[0], pcd);
```

Comment:
`fragment.ply` / `fragment.pcd` / `cloud_bin_0.pcd`, are similar data (slightly different in terms of exact number of points), therefore used `cloud_bin_0.pcd` from `SamplePCDFragment` dataset consistently, and modified the unit test expected results, as required.